### PR TITLE
Tekton create git release and populate with artifacts

### PR DESCRIPTION
**Pull request by Gregory-Pereira**
## Related Issues and Dependencies

Still needs testing, I have provided my example task runs to the pr for use in testing. Can be removed after.

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements
Added in 2 tasks to create a release from an existing tag and then populate that release with artifacts generated

### CODE REVIEW

**Comment by goern** [comment]
looks good!! nice work. I wonder, if there is something from upstream tekton catalog we can reuse?

File: greg/greg-issue-binding.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+apiVersion: triggers.tekton.dev/v1alpha1
```
Comment (harshad16) [inline greg/greg-issue-binding.yaml:1]: drop the greg dir

**Review by harshad16 — CHANGES_REQUESTED** [review]
Follow the comments same as #129 
thanks :100:

**Comment by Gregory-Pereira** [comment]
Opening new PR from feature branch.

**Pull request closed without merging**